src/v/storage/disk_log_impl.cc

```diff
@@ -2161,7 +2161,7 @@ disk_log_impl::offset_range_size(
     size_t current_size = 0;
     // Last offset included to the result, default value means that we didn't
     // find anything
-    model::offset last_included_offset;
+    model::offset last_included_offset = {};
```

Comment:
@WillemKauf can you explain why this would be unitialized?

Reply:
I think I was just confused about the offset's default value (`numeric_limits::min()`) here. Ultimately this doesn't change the behaviour.

Reply:
thanks